src/os/annotation/annotation.js

```diff
@@ -15,7 +15,10 @@ os.annotation.DEFAULT_OPTIONS = {
   showName: true,
   showDescription: true,
   size: [200, 100],
-  offset: [0, -75]
+  offset: [0, -75],
+  headerBG: '#52575c',
+  bodyBG: '#3a3f44'
+
```

Comment:
Can you set the color defaults to the constants you defined later in this file? ie `os.annotation.DEFAULT_ANNOTATION_HEADER_BACKGROUND_COLOR`